recipes/libpqxx/all/conandata.yml

```diff
@@ -38,12 +38,18 @@ sources:
   "7.3.1":
```

Comment:
I believe we can keep last patch of each minor version the next time, WDYT?

Reply:
Maintenance of older versions has not been a burden for me, so I can keep it for consumers using older versions.